addons/account/tests/test_reconciliation.py

```diff
@@ -271,7 +271,7 @@ def test_partial_reconcile_currencies_01(self):
             'company_id': self.company.id,
             'date': time.strftime('%Y') + '-' + '07' + '-01',
             'partner_id': self.partner_agrolait_id,
-            'payment_method_line_id': self.inbound_payment_method_line.id,
+            'payment_method_line_id': self.bank_journal_euro.outbound_payment_method_line_ids[0].id,
```

Comment:
can you please clarify why this change is needed? thanks

Reply:
There are tests that are failing due to the constrains

Here for example the payment journal is `self.bank_journal_id`, so a payment method line of that journal has to be set